osbs/constants.py

```diff
@@ -88,6 +88,9 @@
 # HTTP methods that we should retry on
 HTTP_RETRIES_METHODS_WHITELIST = ['GET', 'PUT', 'POST', 'DELETE']
 
+# requests timeout in seconds
+HTTP_REQUEST_TIMEOUT = 120
```

Comment:
Please can we increase this to something like 600? cf. https://github.com/openshift/origin/blob/v3.6.0/pkg/cmd/server/api/types.go#L716-L718